drupal/sites/all/features/work_and_host_stats/work_and_host_stats.views_default.inc

```diff
@@ -10631,6 +10631,43 @@ else {
       'field' => 'name',
       'relationship' => 'none',
     ),
+    'phpcode_8' => array(
+      'label' => 'dont suppress otuput',
```

Comment:
Thanks, I'll correct it. That being said the label doesn't matter because it's a "hidden" field in a Drupal view, meaning the label is never shown to the user. The only people who would see it would be Drupal admins.